.github/workflows/pull-unit-tests.yml

```diff
@@ -0,0 +1,37 @@
+name: PR Unit Tests
+
+on: 
+  pull_request:
+    types: [opened, edited, synchronize, reopened, ready_for_review]
+    paths:
+      - ".github/workflows/pull-unit-tests.yml"
+      - "backend/**"
+      - "public/**"
+      - "src/**"
+      - "tests/**"
+      - "package.json"
+      - "Makefile"
+      - "Dockerfile.local"
+
+jobs:
+  run-unit-test:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: gardenlinux/workflow-telemetry-action@v2
+        with:
+          comment_on_pr: false
+      - uses: actions/checkout@v4
+        with:
+          fetch-depth: 0
+      - uses: actions/setup-node@v4
+        with:
+          node-version: 20
+      - name: setup_busola
+        shell: bash
+        run: |
+          set -e
+          npm ci
```

Comment:
We can use alias: `clean-install` to be easier to understand what this step is doing.

Reply:
we are using `npm ci` everywhere and it's clear, so I'll go with that to keep the convention on which we agreed, if we want to change that it should be changed in all of the workflows.

Reply:
We can extrend `ci` to `clean-install`, looks ok